tests/p2p/test_protocol.py

```diff
@@ -155,6 +157,7 @@ def test_valid_hello(self):
         self.assertFalse(self.conn.tr1.disconnecting)
         self.assertFalse(self.conn.tr2.disconnecting)
 
+    @pytest.mark.skipif(sys.platform == 'win32', reason='resolver._parseCall not defined on Windows')
```

Comment:
It is used only to remove warnings. Maybe we should do the clean up in a different way or add the if inside the test. As far as I understood, the only consequence is that there will be a warning message after the test is finished.

Reply:
This doesn't affect any warnings, it only prevents the test from running on Windows. The rationale is that all tests that are known to fail will be skipped so `make tests-lib` runs to completion on Windows, and we can then remove the `continue-on-error` on the CI, which will make the task actually fail when a test fails which wasn't known to fail.